tasks/main.yml

```diff
@@ -28,71 +28,12 @@
   ansible.builtin.template:
     src: haproxy.cfg.j2
     dest: /etc/haproxy/haproxy.cfg
-    validate: haproxy -c -f %s
+    # validate: haproxy -c -f %s
     mode: "0640"
+    backup: true
   notify:
     - Restart haproxy
 
-- name: Copy additional haproxy configuration
-  ansible.builtin.copy:
-    src: "{{ item.src }}"
-    dest: "/etc/haproxy/conf.d/{{ item.name }}.cfg"
-    owner: haproxy
-    group: haproxy
-    mode: "0644"
-  notify:
-    - Test Config
-    - Reload haproxy
-  with_items: "{{ haproxy_additional_host_configs }}"
```

Comment:
Adds support for static configurations i.e so we don't only render configurations from consul. The options provided by the base role(from Roberto) doesnt allow for proper configuration customization

Reply:
We don't use this anywhere though because everything is based on consul